tools/m2ctx/m2ctx.py

```diff
@@ -51,20 +63,76 @@ def import_c_file(in_file: Path) -> str:
 
     return out_text
 
+def pcpp_import(in_file: Path) -> str:
+    c_command = ['pcpp', *PCPP_FLAGS, str(in_file)]
+    return try_import(c_command)
```

Comment:
Will implement a more elegant version of this later that uses the API directly. Works really well [as-is](https://decomp.me/scratch/jJl4p), though.